SRS.md

```diff
@@ -142,43 +138,23 @@ This use case covers the default view and contains the existing exams.
-
-[Update Progress Bar](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_UpdatedProgressBar.md)
-
-#### 3.1.6 Send Notifications
+#### 3.1.5 Start/Stop Timer
 
-To inform and acitivate the user to learn, the systems sends push up messages to the users smartphone.
+The estimated time can be count down by a start Timer button. As well the Timer can be stopped by another button.
 
-[Send Notifications](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_SendNotifactions.md)
+[Start / Stop Timer](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_StartStopTimer.md)
 
-#### 3.1.7-3.1.9 Exam
+#### 3.1.6-3.1.9 Exam
 
-There are three use cases for this subitem. These are three of the typical CRUD operations, so create, edit and delete an exam.
+There are four use cases for this subitem. These are three of the typical CRUD operations, so create, edit and delete an exam. Additional you can use a switchbutton to set the selected Exam on 'finished'. This will show an additional label. Also it will effect the sorting on the dashboard.
 
 [Create Exam](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_CreateExam.md)
 
 [Edit Exam](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_EditExam.md)
 
 [Delete Exam](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_DeleteExam.md)
 
-#### 3.1.10-3.1.13 Topic
-
-Since the topics are similar to the exams, the use cases create, edit, delete and show topic should be mentioned here as well.
-
-[Create Topic](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_CreateTopic.md)
-
-[Edit Topic](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_EditTopic.md)
-
-[Delete Topic](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_DeleteTopic.md)
-
-[Show Topic](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_ShowTopic.md)
-
-#### 3.1.14 Agenda View
-
-Here you can make various adjustments for the dashboard. Among other things, you can select how the exams are displayed, and you can access the individual subchapters for the exams. There are also two designs for the light and dark modes. - tbd
+[Finish Exam](https://github.com/nEXam-App/nEXam-doc/blob/7869f8d2f7074c54fab07d40a0b7f2b6814a376a/Use_Case_Specification/use_case_setExamFinished.md)
```

Comment:
Afterwards:
#### 3.1.14 Agenda View

Here you can make various adjustments for the dashboard. Among other things, you can select how the exams are displayed, and you can access the individual subchapters for the exams. There are also two designs for the light and dark modes. - tbd
TOPICS?